WORKSPACE

```diff
@@ -376,6 +376,13 @@ container_pull(
     repository = "kubevirt/alpine-ext-kernel-boot-demo",
 )
 
+container_pull(
+    name = "fedora_realtime",
+    digest = "sha256:8cfba43a0e60f4524391f1576d78fd8e0428ea72b71c8323c7c206359c48849b",
+    registry = "quay.io",
+    repository = "jordigilh/fedora-realtime-container-disk",
```

Comment:
Need to change this to where the final location of the container image will be (kubevirt most probably). I'm pushing this for now to test the validity of the realtime functional test lane.